src/screens/manage-token/manage-token.tsx

```diff
@@ -123,14 +125,22 @@ export class ManageTokenComponent extends React.Component<
         if (blockchainToken) {
             if (blockchainToken.name === '') blockchainToken = undefined;
         }
+
         const symbol = blockchainToken ? blockchainToken.symbol : staticToken.symbol;
         const decimals = blockchainToken ? blockchainToken.decimals : staticToken.decimals;
         const contractAddress = staticToken ? staticToken.contractAddress : this.state.fieldInput;
+
+        let icon: any = staticToken ? { uri: staticToken.logo } : GENERIC_TOKEN_ICON;
+
+        if (String(symbol).toUpperCase() === XSGD_MAINNET.symbol) {
```

Comment:
why do we need this ?

Reply:
Just a pretty stuff to add icon to XSGD on testnet, because on blockchain search we don't have XSGD icon. It's returned the `GENERIC_TOKEN_ICON`